app/views/suggestions/edit.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Edit your suggestion</h1>
+<a href="/users/current_user.id"> Profile </a>
```

Comment:
In an `.erb` file, any Ruby code needs to be placed in ERB tags, for example: `<%= current_user.id %>`